### ISSUE TITLE
Accessibility issue: <svg> elements with an img role must have an alternative text

### ISSUE THREAD

**Issue by nali-github** [adobe/react-spectrum-charts#474]
### Provide a general summary of the issue here

We got accessibility ticket when rendering chart as svg: `<svg> elements with an img role must have an alternative text`
<img width="1708" alt="axe DevTools complains" src="https://github.com/user-attachments/assets/6d41b1dc-3455-4759-8fbc-1dbdb810bf35">

### 🤔 Expected Behavior?

Can add `aria-label` to the components

### 😯 Current Behavior

Unable to add `aria-label` to the components

### 🔦 Context

[Here](https://git.corp.adobe.com/frontend/quarry/blob/main/packages/%40quarry-connected/reporting-dashboard/src/CoreWidgets/RSCBar.tsx#L134) is the implementation. I tried to add a `description` prop to `Chart` level in RSC, it's added to the top div. The `Bar` component doesn't support `description` prop.
 
Based on `axe DevTools`, the aria-label needed to add to 2 lower level elements:
<img width="1247" alt="rsc" src="https://github.com/user-attachments/assets/d7eef0ce-dd64-4ef6-b69a-a9c0d5ce01a6">

### 🖥️ Steps to Reproduce

Storybook link [here](https://experience-stage.adobe.com/#/@cjmstage/sname:prod/quarry/storybook-connected/?path=%2Fstory%2Freporting-dashboard--connected-dashboard-default&args=dashboardId%3Add1221e6449b4029995b0c7411ba9bac%3Bvariant%3Arsc) . 
Scan using the `axe DevTools` can see the problem.

### Version

1.15.0

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Mac
